designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/NodesApiEndpoints.scala

```diff
@@ -76,28 +128,82 @@ class NodesApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEndpoi
       .tag("Nodes")
       .post
       .in("nodes" / path[ProcessName]("scenarioName") / "validation")
```

Comment:
![Screenshot 2024-03-27 at 15 12 41](https://github.com/TouK/nussknacker/assets/231553/dc6ac3c9-53aa-4aff-b08d-112b4a3cc5db)

It looks like examples without the name (or description, IDK) have the "Example{STH}" name in the swagger.